x-pack/plugins/uptime/public/components/overview/synthetics_callout.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiCallOut, EuiSpacer } from '@elastic/eui';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+const SYNTHETICS_CALLOUT_LS_KEY = 'xpack.uptime.syntheticsCallout.display';
+const shouldShowSyntheticsCallout = () => {
+  let value = localStorage.getItem(SYNTHETICS_CALLOUT_LS_KEY);
+  if (value === null) {
+    localStorage.setItem(SYNTHETICS_CALLOUT_LS_KEY, 'true');
+    value = 'true';
+  }
+  return value === 'true';
+};
+const hideSyntheticsCallout = () => localStorage.setItem(SYNTHETICS_CALLOUT_LS_KEY, 'false');
+
+export const SyntheticsCallout = () => {
+  const [shouldShow, setShouldShow] = useState<boolean>(shouldShowSyntheticsCallout());
+  if (!shouldShow) {
+    return null;
+  }
+  return (
+    <>
+      <EuiCallOut
+        title={i18n.translate('xpack.uptime.overview.pageHeader.syntheticsCallout.title', {
+          defaultMessage: 'Elastic Synthetics',
+        })}
+        iconType="beaker"
+      >
+        <p>
+          <FormattedMessage
+            id="xpack.uptime.overview.pageHeader.syntheticsCallout.content"
+            defaultMessage="Elastic Uptime now supports synthetic browser monitors! Learn how to use them {here}."
```

Comment:
@drewpost thoughts on this copy? and the URL below?

Reply:
Can I make a small design suggestion for the callout layout? I would like to suggest that we make the link to the synthetics elastic.co page the primary link and allow for users to dismiss the callout as a secondary option. I don't think the small inline link is going to be clicked, thereby making the dismiss option the most common action. Perhaps something like this? I'm sure that we can beef up the copy as well of this announcement 😊

<img width="1641" alt="Screenshot 2020-10-06 at 09 08 55" src="https://user-images.githubusercontent.com/4104278/95169811-c1638f80-07b3-11eb-8eb1-d46cf3904b1a.png">

Reply:
@andrewvc "Uptime is now previewing support for scripted multi-step availability checks. This means you can interact with elements of a webpage and check the availability of an entire journey (such as making a purchase or signing into a system) instead of just a simple single page up/down check. Please click below to read more and, if you'd like to be one of the first to use these capabilities, you can download our preview synthetics agent and view your synthetic checks in Uptime"

Reply:
@andrewvc https://www.elastic.co/what-is/synthetic-monitoring That's the link